### PR TITLE
5.4.0

### PR DESCRIPTION
**Pull request by billiegoose**
Releases https://github.com/stoplightio/prism/pull/2401 in version 5.4.0

### CODE REVIEW

**Comment by chohmann** [comment]
If you haven't released this yet, could you wait? We have one more PR that is going to get merged today that we'd like to release at the same time.

**Comment by brendarearden** [comment]
https://github.com/stoplightio/prism/pull/2402 specifically this is the PR we would like to have included with this release

**Review by EdVinyard — APPROVED** [review]

**Comment by EdVinyard** [comment]
Whoops!  I looked through the changed files and approved before I saw @chohmann and @brendarearden 's comments above.

**Comment by brendarearden** [comment]
my PR has been merged and it also fixes the lint error

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 # Unreleased
 
+# 5.4.0 (2023.10.09)
+
+- added more validation around optional security. [#2401](https://github.com/stoplightio/prism/pull/2401)
+
```
Comment (chohmann) [inline CHANGELOG.md:14]: Can we add entries for the other two PRs/fixes/features that are being released too?

The other 2 are:
1. Fixed issue with filling additional properties [#2398](https://github.com/stoplightio/prism/pull/2398)
2. Fixed issue with internal refs inside json schemas [#2402](https://github.com/stoplightio/prism/pull/2402)

**Review by chohmann — CHANGES_REQUESTED** [review]
Can we update the changelog to include the other two issues being fixed/released?

**Comment by chohmann** [comment]
I don't know what happened, but it looks like the changes for https://github.com/stoplightio/prism/pull/2398 are no longer on master.

**Comment by billiegoose** [comment]
oh gosh darn it. This repo is set up with merge commits by default instead of squash commits, so `master` got really messy when my PR got merged and I tried to fix it but clearly fucked things up a bit.

**Comment by billiegoose** [comment]
I'm fixing the PR settings to be in line with what `platform-internal` uses.

Meanwhile, I'll cherry-pick https://github.com/stoplightio/prism/pull/2398 back somehow. Or if anyone has a older copy of `master` and wants to force push, go ahead.

**Comment by billiegoose** [comment]
@chohmann are there any other PRs that got merged since the 5.3.2 release that are missing?

**Comment by chohmann** [comment]
>@chohmann are there any other PRs that got merged since the 5.3.2 release that are missing?

@billiegoose I think that's the only one.

The three PRs that should be released with 5.4.0 are:
1. https://github.com/stoplightio/prism/pull/2398
2. https://github.com/stoplightio/prism/pull/2401
3. https://github.com/stoplightio/prism/pull/2402

**Comment by billiegoose** [comment]
Alright, 🤣 hopefully it's good to go now. Sorry about the mishap folks, won't happen again (since I've disabled all other merge options mwahahaha).

File: CHANGELOG.md (hunk 1 above)
Comment (billiegoose) [reply CHANGELOG.md]: done!

**Comment by chohmann** [comment]
@billiegoose awesome! I think everything looks good, but I see that the version as been updated lerna.json and package.json's, but the tag has not been created in github. That happens when running yarn release usually.

Are you following the release steps documented [here](https://meta.stoplight.io/docs/handbook/1516eef042cd8-releasing-a-new-version-of-prism)?

**Comment by billiegoose** [comment]
No, I scanned the CONTRIBUTING.md and didn't find anything so I just did my best copy-paste.

I gotta run to a dinner, any chance someone else can carry this across the finish line?

**Comment by chohmann** [comment]
I got it, @billiegoose !

**Pull request closed without merging**